.gitignore

```diff
@@ -1,5 +1,6 @@
 /build
 /vendor
+vendor-bin/**/composer.lock
```

Comment:
missing `vendor-bin/**/vendor` - otherwise that part will get accidentally committed

Reply:
fixed
I also fixed it in password_policy. It is so difficult to get one app as "gold" that can be copied from!